app/scripts/components/exploration/types.d.ts.ts

```diff
@@ -67,10 +74,11 @@ export interface TimelineDatasetAnalysisError {
   error: any;
   meta: Partial<AnalysisMeta>;
 }
+export type TimeseriesData = Record<string, AnalysisTimeseriesEntry[]>;
```

Comment:
Because CMR timeseries data can have multiple bands, I changed the type of TimeseriesData this way. (And made the changes requires for UI.)